config/application.rb

```diff
@@ -47,7 +47,10 @@ class Application < Rails::Application
       Sequel.extension(:core_extensions, :postgres_schemata)
       Sequel::Model.db.extension(:pg_array, :pg_inet)
     end
-
+    # Configure Sequel to use the ShardedThreadedConnectionPool
+    # so that we can close connections on-demand.
+    config.sequel.servers = {}
```

Comment:
This comment needs to be expounded a bit to explain that:
- The default connection pool does not support closing connections
- We must be able to close connections on demand to clear the connection cache after policy loads (perhaps link to this PR in the comment as well)
- The ShardedThreadedConnectionPool does support closing connections on-demand (perhaps link to the [docs here](https://www.rubydoc.info/github/jeremyevans/sequel/Sequel/ShardedThreadedConnectionPool))
- Sequel is configured to use the ShardedThreadedConnectionPool by setting the `servers` configuration on the database connection (perhaps link to the [docs here](https://github.com/jeremyevans/sequel/blob/master/doc/sharding.rdoc#the-servers-database-option-))